graphene-writer/src/main/resources/application-local.yml

```diff
@@ -118,9 +121,12 @@ graphene:
               templateIndexPattern: "index-based-key-path*"
               index: "index-based-key-path"
               type: "path"
+              cacheExpireIntervalInSeconds: 300
+              targetProcessTime: 30000
               bulk:
                 actions: 10000
-                interval: 500
+                preloadActions: 5000
+                interval: 10000
```

Comment:
How about renaming those settings to make them more readable? The term `PreLoad` simply feels like loading a specific key to memory in advance. Rather, it would be better to change the `PreIndex` or another name that suits the actual operation.